kinde_fastapi/examples/management_token_example.py

```
@@ -0,0 +1,119 @@
+from fastapi import FastAPI, Depends, HTTPException, status
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+import os
+import requests
```

Ruff (0.12.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)